src/content/docs/ja/guides/deploy/aws.mdx

```diff
@@ -278,3 +222,10 @@ AWSで継続的デプロイを設定する方法はたくさんあります。Gi
     :::note
     `BUCKET_ID`はS3バケットの名前です。`DISTRIBUTION_ID`はCloudFrontのディストリビューションIDです。CloudFrontのディストリビューションIDは**CloudFront > ディストリビューション > ID**で確認できます。
     :::
+</Steps>
+
+## コミュニティリソース
+
+- [Deploy Astro to AWS Amplify](https://www.launchfa.st/blog/deploy-astro-aws-amplify)
+- [Deploy Astro to AWS Elastic Beanstalk](https://www.launchfa.st/blog/deploy-astro-aws-elastic-beanstalk)
+- [Deploy Astro to Amazon ECS on AWS Fargate](https://www.launchfa.st/blog/deploy-astro-aws-fargate)
```

Comment:
Thanks @magurotabetai, this looks good but we're missing translation for these links here - could you update this PR with this part translated? 🙌